xml/obs_multibuild.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC
+  "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.docbook.org/xml/4.5/docbookx.dtd"
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+<chapter id="cha.obs.multibuild">
+ <title>Multibuild</title>
+ <section>
+  <title>Overview</title>
+  <para>
+   Multibuild is used for those sources that can be used to build more than a
```

Comment:
Maybe this?

"Multibuild is used for sources that can build multiple packages."